index.html

```diff
@@ -0,0 +1,5 @@
+<html>
+    <body>
+        RAMPER PROJECT
```

Comment:
please change the name to be more descriptive

Reply:
ok